src/tempfile.rs

```diff
@@ -0,0 +1,167 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+//! Create temporary files.
```

Comment:
nit: Can you re-word this a bit so is similar to the other documentation entries?

Some examples:
- Structures, helpers, and type definitions for working with errno.
- Enum and function for dealing with an allocated disk space by fallocate.
- Facilities for sending log message to syslog.